docs/source/building/building.rst

```diff
@@ -76,6 +80,11 @@ The dependencies can be installed via the package manager
 
 (in new terminals, re-activate the environment with ``spack env activate hipace-dev`` again)
 
+.. note::
+   On Ubuntu distributions, the InstallErrror ``"OpenMPI requires both C and Fortran compilers"`` can occur because the Fortran compilers are sometimes not set automatically in Spack.
+   To fix this, the Fortran compilers must be set manually using ``spack config edit compilers`` (more information can be found `here <https://spack.readthedocs.io/en/latest/getting_started.html#compiler-configuration>`__).
+   For GCC, the flags ``f77 : null`` and ``fc : null`` must be set to ``fcc : gfortran`` and ``fc : gfortran``.
```

Comment:
Are you supposed to change `f77` to `fcc` or is this a typo?

Reply:
That is a typo!